docs/channel.md

```diff
@@ -71,10 +71,6 @@ For example, the `Channel.of()` factory can be used to create a channel from an
 Channel.of(1, 2, 3).view()
 ```
 
-:::{versionadded} 20.07.0
-`channel` was introduced as an alias of `Channel`, allowing factory methods to be specified as `channel.of()` or `Channel.of()`, and so on.
```

Comment:
Do we want to completely remove reference to this? It's out there in the wild already..

Reply:
we could take it or leave it. the strict parser will recognize it and give a deprecation warning

Reply:
I agree with Phil here that it's already out in the wild and will cause frustration if it's not documented why.

Maybe:

`channel` was introduced as an alias of `Channel`, allowing factory methods to be specified as `channel.of()` or `Channel.of()`, and so on. It is preferable to use `Channel` to maintain clarity and follow best practices. `channel` will be depreciated in the future.

Or something like that.